### ISSUE TITLE
[Android] UnsatisfiedLinkError: couldn't find DSO to load: libstatic-webp.so is 64-bit instead of 32-bit

### ISSUE THREAD

**Issue by carstenhag** [WhatsApp/stickers#293]
**Specifications**  
OS Level: Lollipop 5.1
Device Manufacturer: One Plus 2
Native platforms: armeabi-v7a, armeabi, arm64-v8a

**Stickers**
My stickers do comply with the sticker requirements (512x512 and < 100 KB): Yes
My tray icon does comply with the requirements (96x96 and < 50 KB): Yes
I am using [WebP/PNG] images for stickers. Yes

**The problem**  
The user said this:  
>Can't open it. It loads something and just closes down before anything shows. Just a blank white screen for half a second.

See attached logcat. I think there's an issue with the `libstatic-webp.so` file.

**How anyone may be able to reproduce the problem**  
Pretty much install the app and start it, I guess.

**Links**  
https://play.google.com/store/apps/details?id=de.chagemann.thinkingstickers  
https://gitlab.com/carstenhag/thinking-stickers

**Comment by carstenhag** [comment]
[Log_2018-11-22_18-25-58.txt](https://github.com/WhatsApp/stickers/files/2608803/Log_2018-11-22_18-25-58.txt)

**Comment by carstenhag** [comment]
Somewhat similar issue (64-bit instead of 32-bit) here: https://github.com/facebook/react-native/issues/11027

Note: All webp images were generated with Android Studio.

**Comment by pranshu1701** [comment]
Looks like a fresco issue

**Comment by Vengat121** [comment]
having the same issue, is this issue gets solved?

**Comment by savvisingh** [comment]
Any fixes?

**Comment by carstenhag** [comment]
@Vengat121 @savvisingh Hey, if you have the same issue you can also +1 the original post. That way, the repo mantainers have it easier prioritizing what to fix.

**Comment by AhmedNader65** [comment]
Same issue

**Comment by simonzhexu** [comment]
working on it, but not an easy fix.

**Comment by YamilSeo** [comment]
Do I have a solution for the same problem?

**Comment by simonzhexu** [comment]
I added a fix that supports App Bundle, and upgraded fresco. Are you still seeing this problem?

**Comment by rebelholic** [comment]
I already tested, it's worked after fixed from you. Thanks

**Comment by carstenhag** [comment]
https://github.com/WhatsApp/stickers/commit/00c372ea5e2534ff9279c7103a321a4212b0f579

**Comment by carstenhag** [comment]
Nevermind my comment, figured it out. Look at the referenced issue for the solution: Use gradle 4.10.1 so instant app still works

**Comment by simonzhexu** [comment]
Awesome, Thank you guys! Closing the issue for now. Reopen if needed.

**Comment by hardapps1692** [comment]
I am facing this issue daily in my stickers apps. I have updated the dependency with new version 1.13.0 still its crashing in all 64 bits devices. How to resolve it? Please help me out

**Comment by MarvinJWendt** [comment]
Hi @hardapps1692, please make sure that you use the latest version of the repository.
If you already do, or it doesn't solve your problem please open a new Issue (since this is kinda old) and mention `#293` in it.

**Comment by hardapps1692** [comment]
> Hi @hardapps1692, please make sure that you use the latest version of the repository.
> If you already do, or it doesn't solve your problem please open a new Issue (since this is kinda old) and mention `#293` in it.

If I will open a new issue then I will get only comments no any bug fix or suggestion. So I would like to continue issue with this thread.

**Comment by MarvinJWendt** [comment]
> Nevermind my comment, figured it out. Look at the referenced issue for the solution: Use gradle 4.10.1 so instant app still works

@hardapps1692 The author of this thread has resolved his issue. If we continue discussing here, he will get a notification for each comment, which would be annoying for him. 
If you open a new Issue, we can discuss your problem there without notifying people that don't want to be involved. 

Also we can help you a lot faster if you provide the information that is requested in the `new issue` template. Debug logs would be perfect :)

Since this Issue is 4 Months old, your problem could have a completely different reason and/or solution.
